### PR TITLE
docs & examples update

### PR DESCRIPTION
**Pull request by JosepSampe**
- docs updated
- executor of all the examples updated
- changelog updated
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/api-details.md [hunk 1]
```diff
@@ -27,37 +30,38 @@ The available calls within an executor are:
...
-Initialize and return an IBM Cloud Functions executor object.
+Initialize and return Localhost executor object.
 
 |Parameter | Default | Description|
 |---|---|---|
 |config | None | Settings passed in here will override those in lithops_config|
 |runtime |  None | Name of the docker image to run the functions |
-|runtime_memory | 256 | Memory (in MB) to use to run the functions |
-|storage_backend | ibm_cos | Storage backend to store temp data|
+|workers | cpu_count | Max number of parallel workers |
```
Comment (gilv) [inline docs/api-details.md:41]: @JosepSampe workers = cpu_count? Why did you removed runtime_memory?

File: docs/api-details.md [hunk 2]
```diff
@@ -27,37 +30,38 @@ The available calls within an executor are:
...
+|storage | localhost | Storage backend to store temp data|
 |rabbitmq_monitor | False | Activate RabbitMQ monitoring |
 |log_level | None | Log level printing (INFO, DEBUG, ...) |
-|remote_invoker | False | Spawn a function that will perform the actual job invocation (True/False) |
```
Comment (gilv) [inline docs/api-details.md:42]: @JosepSampe why it was removed?

File: docs/api-details.md (hunk 1 above)
Comment (JosepSampe) [reply docs/api-details.md:41]: @gilv localhost and standalone backends has no `runtime_memory` param. `runtime_memory `is specific param of serverless backends

Comment (JosepSampe) [reply docs/api-details.md:41]: @gilv localhost and standalone backends has cpu_count as max workers

File: docs/api-details.md (hunk 2 above)
Comment (JosepSampe) [reply docs/api-details.md:42]: @gilv localhost and standalone backends has no remote_invoker feature

**Comment by gilv** [comment]
@JosepSampe thanks. I will submit other changes to docs on top of this. Let's merge it.

**Pull request merged**